style.css

```diff
@@ -508,4 +508,73 @@ main > div img {
  div.evidon-notice-link {
    display:none;
  }
- 
\ No newline at end of file
+ 
+  /*
+  * Styles for Dropdown Region Selector
+  */
+.region-dropdown {
+  border: 1px solid #d3d3d3;
+  background-color: #ffffff;
+  border-radius: 4px;
+  display: inline-block;
+  font-size: inherit;
+  padding: 3px 10px 5px 8px;
+  margin: 0;
+  visibility: hidden;
+  position: absolute;
+  z-index: 0;
+}
+
+.region-dropdown.visible {
+  visibility: visible;
+  z-index: 2;
+}
+
+.region-dropdown ul {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+a.region-dropdown-picker {
+  color: #505050;
+  font-size: .9rem;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  cursor: pointer;
+  padding-right: 5px;
+}
+
+a.region-dropdown-picker:focus:not(.focus-visible),
+a.region-dropdown-picker:active {
+  outline: none;
+}
+
+a.region-dropdown-picker:focus-visible {
```

Comment:
Not sure if `focus-visible` is added automatically when you tab or you need to implement it.

Reply:
Changed this one, added .focus for the dropdownLinks when it's visible